CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+
+## Reporting Bugs
+If you discover a bug, first check the [GitHub Issues page](https://github.com/sashabaranov/go-openai/issues) to see if the issue has already been reported. If you're reporting a new issue, please use the "Bug report" template and provide detailed information about the problem, including steps to reproduce it.
+
+## Suggesting Features
+If you want to suggest a new feature or improvement, first check the [GitHub Issues page](https://github.com/sashabaranov/go-openai/issues) to ensure a similar suggestion hasn't already been made. Use the "Feature request" template to provide a detailed description of your suggestion.
+
+## Reporting Vulnerabilities
+If you identify a security concern, please use the "Report a security vulnerability" template on the [GitHub Issues page](https://github.com/sashabaranov/go-openai/issues) to share the details. This report will only be viewable to repository maintainers. You will be credited if the advisory is published.
+
+## Questions for Users
+If you have questions, please utilize [StackOverflow](https://stackoverflow.com/) or the [GitHub Discussions page](https://github.com/sashabaranov/go-openai/discussions).
+
+## Contributing Code
+There might already be a similar pull requests submitted! Please search for [pull requests](https://github.com/sashabaranov/go-openai/pulls) before creating one.
+
+### Requirements for Merging a Pull Request
+
+The requirements to accept a pull request are as follows:
+
+- Features not provided by the OpenAI API will not be accepted.
+- The functionality of the feature must match that of the official OpenAI API.
+- All pull requests should be written in Go according to common conventions, formatted with `goimports`, and free of warnings from tools like `golangci-lint`.
+- Include tests and ensure all tests pass.
+- Maintain test coverage without any reduction.
+- All pull requests require approval from at least one Go OpenAI maintainer.
+
+**Note:**  
+The merging method for pull requests in this repository is squash merge.
+
+### Creating a Pull Request
+- Fork the repository.
+- Create a new branch and commit your changes.
+- Push that branch to GitHub.
+- Start a new Pull Request on GitHub. (Please use the pull request template to provide detailed information.)
+
+**Note:**  
+If your changes introduce breaking changes, please prefix your pull request title with "[BREAKING_CHANGES]".
+
+### Code Style
+In this project, we adhere to the standard coding style of Go. Your code should maintain consistency with the rest of the codebase. To achieve this, please format your code using tools like `goimports` and resolve any syntax or style issues with `golangci-lint`.
+
+**Run goimports:**
+```
+go install golang.org/x/tools/cmd/goimports@latest
+```
+
+```
+goimports -w .
```

Comment:
This is a really great PR, thank you so much @vvatanabe! Sorry for such a long delay with the review.

Just a note: it seems that golangci-lint does include goimports into itself, so it's enough to run to just run the golangci-lint.

To reproduce:

```diff
diff --git a/completion_test.go b/completion_test.go
index 844ef48..3a63504 100644
--- a/completion_test.go
+++ b/completion_test.go
@@ -4,6 +4,7 @@ import (
        . "github.com/sashabaranov/go-openai"
        "github.com/sashabaranov/go-openai/internal/test/checks"

+       "testing"
        "context"
        "encoding/json"
        "errors"
@@ -12,7 +13,6 @@ import (
        "net/http"
        "strconv"
        "strings"
-       "testing"
        "time"
 )
```

Produces:

```
% golangci-lint run ./...
completion_test.go:7: File is not `goimports`-ed (goimports)
	"testing"
```

Reply:
@sashabaranov Thanks for the feedback. Indeed, `golangci-lint` does give warnings, but to fix them, you need to run goimports yourself. I've added instructions on how to run `goimports` to help developers who are not familiar with Go handle it smoothly.

---
Don't worry about the delayed response. You should focus on what you need to prioritize! It's more important.